lib/flutter_isolate.dart

```diff
@@ -100,7 +100,7 @@ class FlutterIsolate {
   static get current => _current != null ? _current : FlutterIsolate._();
   static void _isolateInitialize() {
     WidgetsFlutterBinding.ensureInitialized();
-    window.onPlatformMessage = BinaryMessages.handlePlatformMessage;
+    window.onPlatformMessage = ServicesBinding.instance.defaultBinaryMessenger.handlePlatformMessage;
 
```

Comment:
Looking into it further, I'm pretty sure that `WidgetsFlutterBinding.ensureInitialized()` already sets this up so we can probably delete line 103.